pyqtgraph/graphicsItems/HistogramLUTItem.py

```diff
@@ -25,177 +26,268 @@
-    rgbHistogram     (bool) Sets whether the histogram is computed once over all
-                     channels of the image, or once per channel.
-    levelMode        'mono' or 'rgba'. If 'mono', then only a single set of
-                     black/white level lines is drawn, and the levels apply to
-                     all channels in the image. If 'rgba', then one set of
-                     levels is drawn for each channel.
-    gradientPosition 'right' (default) OR 'left'. Which side of the histogram to
-                     put the LUT gradient.
-    ================ ===========================================================
+
+    Parameters
+    ----------
+    image : pyqtgraph.ImageItem, optional
+        If provided, control will be automatically linked to the image and changes to
+        the control will be reflected in the image's appearance. This may also be set
+        via :meth:`setImageItem`.
+    fillHistogram : bool, optional
+        By default, the histogram is rendered with a fill. Performance may be improved
+        by disabling the fill. Additional control over the fill is provided by
+        :meth:`fillHistogram`.
+    levelMode : str, optional
+        'mono'
+            Only a single set of black/white level lines is drawn and the levels apply
+            to all channels in the image.
+        'rgba'
+            One set of levels is drawn for each channel.
+    gradientPosition : str, optional
+        Position of the gradient editor relative to the histogram, either 'right' or
+        'left'.
+
+    Attributes
+    ----------
+    sigLookupTableChanged : signal
+        Emits the HistogramLUTItem itself when the gradient changes
+    sigLevelsChanged : signal
+        Emits the HistogramLUTItem itself while the movable region is changing
+    sigLevelChangeFinished : signal
+        Emits the HistogramLUTItem itself when the movable region is finished changing
+
+    See Also
+    --------
+    :class:`~pyqtgraph.ImageItem`
+        HistogramLUTItem is most useful when paired with an ImageItem.
+
+    :class:`~pyqtgraph.ImageView`
+        Widget containing a paired ImageItem and HistogramLUTItem.
+
+    :class:`~pyqtgraph.HistogramLUTWidget`
+        QWidget containing a HistogramLUTItem for widget-based layouts.
     """
-    
+
     sigLookupTableChanged = QtCore.Signal(object)
     sigLevelsChanged = QtCore.Signal(object)
     sigLevelChangeFinished = QtCore.Signal(object)
-    
-    def __init__(self, image=None, fillHistogram=True, rgbHistogram=False, levelMode='mono', gradientPosition='right'):
+
+    def __init__(self, image=None, fillHistogram=True, levelMode='mono',
+                 gradientPosition='right', orientation='vertical'):
         GraphicsWidget.__init__(self)
         self.lut = None
         self.imageItem = lambda: None  # fake a dead weakref
         self.levelMode = levelMode
-        self.rgbHistogram = rgbHistogram
```

Comment:
`git blame` indicates this was added but never implemented

Reply:
that would be a cool feature tho!

Reply:
I think `levelMode='rgba'` does what this was probably intended to mean. Check `examples/HistogramLUT.py` which has a toggle to switch between mono/rgba.

Reply:
oh cool, it is implemented there, that looks really slick